e2e/pages/events/addHearingBookingDetailsEvent.page.js

```diff
@@ -1,4 +1,5 @@
 const { I } = inject();
+const judgeAndLegalAdvisor = require('../../fragments/judgeAndLegalAdvisor');
```

Comment:
I tend to name these with 'Fragment' suffix e.g. `judgeAndLegalAdvisorFragment`.

Having `judgeAndLegalAdvisor` looks bit wired to me because it looks a bit like a something that represents person (judge and / or legal advisor).

I am curious your opinion.

Reply:
Good idea - this was an existing fragment used in other tests already so I did not alter the name but in future I agree that we could suffix these with 'Fragment'